packages/theming/apple-theme/README.md

```diff
@@ -0,0 +1,10 @@
+# Apple-theme
+
+Code and definitions for creating an Apple Theme for FluentUI React Native. Currently only implemented for macOS.
+
+Some heuristics followed in this theme:
+
+- For most tokens, we default to prefer the apple system colors, using the FluentUI Apple Palette where it makes sense. These mappings are subject to change as we increment on the design.
+- On macOS, there tends to not be a "hover" state for components such as Button, so those tokens are mapped to be identical to the rest state tokens.
```

Comment:
Maybe to make this a little cleaner we can have a macOS and iOS (currently empty) section of platform specific information

Reply:
Most of the iOS information would be the same  (no focus state, prefer system colors), so I'll leave that as a task for when we add the iOS theme